### PR TITLE
add bean discovery mode to beans.xml from documentation

### PR DESCRIPTION
**Pull request by bennetelli**
…lts in an error if it's not present

### CODE REVIEW

File: documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -55,8 +55,8 @@ Finally, you need an _empty_ `beans.xml`. Copy the following content to `src/mai
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="
       http://xmlns.jcp.org/xml/ns/javaee
-      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd">
-
+      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
+  bean-discovery-mode="annotated">
```
Comment (Ladicek) [inline documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc:59]: It is amusing, the previous line says "_empty_ `beans.xml`" :-)

I agree with recommending annotated, for sure.

**Comment by bennetelli** [comment]
So is there anything for me to do here? Or is it your turn? :-)

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Thanks!

**Comment by cescoffier** [comment]
Created Backports: 
- #1156 to [backport-#1155-to-3.x](https://github.com/smallrye/smallrye-reactive-messaging/tree/3.x)

**Pull request merged**